README.md

```diff
@@ -1,150 +1,102 @@
-
-## Collection maintenance
-
-The current maintainers are listed in the [MAINTAINERS](MAINTAINERS) file. If you have questions or need help, feel free to mention them in the proposals.
-
-To learn how to maintain/become a maintainer of this collection, refer to the [Maintainer guidelines](https://docs.ansible.com/ansible/devel/community/maintainers.html).
-
-It is necessary for maintainers of this collection to be subscribed to:
-
-* The collection itself (the `Watch` button -> `All Activity` in the upper right corner of the repository's homepage).
-* The [news-for-maintainers repository](https://github.com/ansible-collections/news-for-maintainers).
-
-They also should be subscribed to Ansible's [The Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn).
-
-## Governance
-
-<!--Describe how the collection is governed. Here can be the following text:-->
-
-The process of decision making in this collection is based on discussing and finding consensus among participants.
+## Included content
 
-Every voice is important. If you have something on your mind, create an issue or dedicated discussion and let's discuss it!
+Click on the name of a role to view that content's documentation:
 
-## Tested with Ansible
+<!--start collection content-->
+### Roles
+Name | Description
+--- | ---
+[cloud.vmware_ops.provision_vcenter](https://github.com/ansible-collections/cloud.vmware_ops/blob/main/roles/provision_vcenter/README.md)|A role to provison Vcenter on ESXi host.
+[cloud.vmware_ops.provision_vm](https://github.com/ansible-collections/cloud.vmware_ops/blob/main/roles/provision_vm/README.md)|A role to create/manage virtual machines on VMware.
+[cloud.vmware_ops.system_settings](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/roles/system_settings/README.md)|A role to manage system settings on VMware.
 
-<!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
+### Playbooks
+Name | Description
+--- | ---
+[cloud.vmware_ops.provision_vcenter](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/playbooks/provision_vcenter/provision_vcsa_on_esxi.yml)|A playbook to provison Vcenter on ESXi host.
+[cloud.vmware_ops.provision_vm](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/playbooks/provision_vm/manage_vm.yml)|A playbook to create/manage virtual machines on VMware.
+[cloud.vmware_ops.system_settings](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/playbooks/system_settings.yml)|A playbook to manage system settings on VMware.
+<!--end collection content-->
 
-## External requirements
+## Installation and Usage
 
-<!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
+### Requirements
```

Comment:
I think it's worth to ada: 
vmware.vmware_rest

python packages:
pyvmomi

and maybe assign a section on EE, what do you think?

Reply:
`vmware.vmware_rest` is mentioned. Not sure if we should specify dependencies of dependency, in that case we would need to specify also `http-asyncio` etc.

> and maybe assign a section on EE, what do you think?

You mean to document EE? We can add it, but it's not needed as part of validation process. So let's add it later.

Reply:
Also I see other collections don't metion the dependency collection dependencies as well:
https://github.com/redhat-cop/cloud.gcp_ops?tab=readme-ov-file#requirements